modules/db/src/main/scala/gem/dao/composite/ProperMotion.scala

```diff
@@ -28,7 +28,8 @@ private object ProperMotionCompositeLemmas {
   import RadialVelocityMeta._
 
   // The parallax member is an angle, which we will store as signed milliarcseconds
-  private implicit lazy val AngleMasMeta: Meta[Angle] =
+  // N.B. if this is marked private we get a spurious "unused" warning
+  implicit lazy val AngleMasMeta: Meta[Angle] =
```

Comment:
I've seen this naming style a couple of times lately for typeclass instances.  I kind of like it better than just `AngleMeta` or `MetaAngle`.  Maybe we should do it everywhere else?

Reply:
I don't feel strongly about it, but we do run the risk of the names getting stale if we do this … I have only been doing it for non-canonical instances (which we only use locally).

Reply:
I just realized I misread the name as `AngleHasMeta`.  I'd seen `Is` used in other places as in `WuzzleIsFunctor`.